backend/.env

```diff
@@ -0,0 +1 @@
+PORT=3000
```

Comment:
?

Reply:
My bad. I have reset the head and pushed without the .env

Reply:
Let's introduce a new step in our best practices that involves reviewing File Changes in Pull Requests (PRs). It's become noticeable that there are file changes being pushed that aren't part of the designated PR. Adding an image for clarification might be helpful. It's crucial to remember: when adding, committing, pushing, or creating a PR, ensure to review the PR changes meticulously. Using the provided checklist is fundamental to guarantee that only intended changes are pushed to the repository.

Reply:
<img width="1119" alt="Screen Shot 2024-01-08 at 10 30 45" src="https://github.com/jacekroszkowiakdev/products-component/assets/42832783/14d3464f-d8c5-4c42-a541-0c1795f41d2a">
I mean, to add an item which include the step of check GitHub "filesChanges", using also an image in this step to describe the step.

Reply:
I understand now. I added some extra information for the git diff command and information on how to use the Source Control in VS Code with screenshots.